### PR TITLE
Fd 322 executed page after bulk operations fix

### PR DESCRIPTION
**Pull request by filipvnencak**

### CODE REVIEW

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 1]
```diff
@@ -145,12 +145,31 @@ const ExecutedWorkflowList = () => {
     setPagination((prev) => ({ ...prev, page: pageNumber }));
   };
 
+  const onSuccessfullDelete = () => {
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:148]: handleOnSuccessfullDelete - as naming conventions suggests

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 2]
```diff
@@ -145,12 +145,31 @@ const ExecutedWorkflowList = () => {
...
+    fetchData();
+  };
+
   return (
     <PageContainer>
       <ExecutedWorkflowBulkOperationsBlock
         workflowsAmount={workflows.result.totalHits}
         selectedWorkflows={state.selectedWorkflows}
         selectAllWorkflows={selectAllWorkflows}
+        onSuccessfullDelete={onSuccessfullDelete}
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:172]: rename function by the handler name

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 3]
```diff
@@ -145,12 +145,31 @@ const ExecutedWorkflowList = () => {
...
+        state.sortBy,
+        state.sortOrder,
+        state.isFlat,
+      );
+
+      setWorkflows(executedWorkflows);
+      setPagination((prev) => ({ ...prev, pageCount: Math.ceil(executedWorkflows.result.totalHits / prev.pageSize) }));
+    };
+    fetchData();
+  };
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:164]: this whole logic should be only once in component - as by DRY (don't repeat yourself) convention

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx (hunk 1 above)
Comment (MarcoMruz) [reply packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:148]: these are our conventions https://github.com/FRINXio/frinx-javascript

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 4]
```diff
@@ -48,24 +48,24 @@ const ExecutedWorkflowList = () => {
...
+  const fetchData = async () => {
+    const executedWorkflows = await getWorkflows(
+      state.workflowId,
+      state.labels,
+      (pagination.page - 1) * pagination.pageSize,
+      pagination.pageSize,
+      state.sortBy,
+      state.sortOrder,
+      state.isFlat,
+    );
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:60]: move this function out of component
- should return executed workflows

then you will call this function in handleOnSuccessfullOperation and also in useEffect (the one where fetchData function is). then you can set returned executedWorkflows with setPagination and setWorkflows functions

**Review by MarcoMruz — CHANGES_REQUESTED** [review]

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 5]
```diff
@@ -37,6 +38,18 @@ const initialState: StateProps = {
   sortOrder: 'DESC',
   labels: [],
 };
+const fetchData = async (state: StateProps, pagination: PaginationState) => {
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:40]: rename to fetchExecutedWorkflows

**Review by MarcoMruz — CHANGES_REQUESTED** [review]

**Review by soson — APPROVED** [review]

**Pull request closed without merging**